### PR TITLE
Adding rtol and atol for pinv and nullspace

### PR DESCRIPTION
**Pull request by sam0410**
This is as discussed in the PR  #29926

### CODE REVIEW

File: stdlib/LinearAlgebra/src/dense.jl [hunk 1]
```diff
@@ -1201,13 +1201,14 @@ factorize(A::Transpose) = transpose(factorize(parent(A)))
 ## Moore-Penrose pseudoinverse
 
 """
-    pinv(M[, rtol::Real])
+    pinv(M, atol::Real, rtol::Real)
```
Comment (andyferris) [inline stdlib/LinearAlgebra/src/dense.jl]: Should this be keyword args?

File: stdlib/LinearAlgebra/src/dense.jl [hunk 2]
```diff
@@ -1285,11 +1285,12 @@ end
 ## Basis for null space
 
 """
-    nullspace(M[, rtol::Real])
+    nullspace(M, atol::Real, rtol::Real)
```
Comment (andyferris) [inline stdlib/LinearAlgebra/src/dense.jl]: Same here

**Review by andyferris — COMMENTED** [review]
I like this idea!

File: stdlib/LinearAlgebra/src/dense.jl (hunk 1 above)
Comment (sam0410) [reply stdlib/LinearAlgebra/src/dense.jl]: Sorry for the delay @andyferris. I resolved this now. I have some issues regarding the test/pinv tests. I will try to resolve them soon.

**Comment by sam0410** [comment]
Hi @andyferris, Sorry for the delay. I resolved your comment. Also there is some issue with the test/pinv tests. I will resolve them soon.

**Comment by sam0410** [comment]
Hi @andyferris, can you please review this PR now? 
Thank you.

**Comment by simonbyrne** [comment]
@sam0410 any chance you could update this along the lines of #29926? Also, add a `NEWS.md` item.

**Comment by sam0410** [comment]
Hi @simonbyrne, I made the changes.
Thanks.

**Comment by sam0410** [comment]
Oh ! Sorry, I didn't make that part clear.

**Comment by simonbyrne** [comment]
No problem, I figured it was easier for me to make the changes than keep asking you for them. Thanks!

**Comment by simonbyrne** [comment]
This can be backported to 1.1 since it also contains the NEWS item for #29926.

**Pull request merged**